### PR TITLE
1 pic, and a few txt updates

### PR DESCRIPTION
**Pull request by kcameron81**
Let'd do this :-)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,16 +94,16 @@
...
 			</section>
-			<section data-markdown data-background-image="assets/class.jpg" data-background-opacity="0.3">
-				<textarea data-template>
-						## Keep learning
-				</textarea>
-			</section>
 			<section data-markdown data-background-image="assets/stamina.png" data-background-opacity="0.3">
 				<textarea data-template>
-						## Keep pushing for change
+						## Kaizen + Failure + Stamina = Success
```
Comment (calshius) [inline index.html]: Try adding another # to this to make it all fit in one line

Comment (kcameron81) [reply index.html]: Makes sense, i've made change and will raise another pull request

**Comment by kcameron81** [comment]
Should be updated to address the reducing font size for Kaizen row, let know what you think bud...i'm learning :-)

File: index.html [hunk 2]
```diff
@@ -101,9 +101,14 @@
 			</section>
 			<section data-markdown data-background-image="assets/stamina.png" data-background-opacity="0.3">
 				<textarea data-template>
-						## Keep pushing for change
+						### Kaizen + Failure + Stamina = Success
 				</textarea>
 			</section>
+			<section data-markdown data-background-image="assets/edconf.jp2" data-background-opacity="0.3">
```
Comment (calshius) [inline index.html:107]: Pitcure isn't showing it seems

**Pull request closed without merging**